src/dbnode/storage/stats/query_stats.go

```diff
@@ -118,32 +177,55 @@ func (q *queryStats) Start() {
 	if q == nil {
 		return
 	}
+	opts := q.tracker.Options()
+	docsTicker := time.NewTicker(opts.MaxDocsLookback)
+	bytesTicker := time.NewTicker(opts.MaxBytesReadLookback)
 	go func() {
-		ticker := time.NewTicker(q.tracker.Lookback())
-		defer ticker.Stop()
+		defer docsTicker.Stop()
+		defer bytesTicker.Stop()
```

Comment:
nit: i think this may be a bit cleaner as
```golang
for {
 select {
   // ..
   case <-q.stopCh:
     break
  }
}
docsTicker.Stop()
bytesTicker.Stop()
return 
```